src/handlers/zapierTriggers.ts

```diff
@@ -0,0 +1,20 @@
+import { fetchStatuses } from './validators';
+import ky from "ky";
+
+export async function zapierTriggers(event: Event) {
+    await sendValidatorStatuses();
+}
+
+async function sendValidatorStatuses() {
+    const statuses = await fetchStatuses();
+
+    try {
+        const res = await ky.post(ZAPIER_WEBHOOK_URL, {
+            body: JSON.stringify(statuses)
+        }).json()
```

Comment:
Dumb question, but do you need `ky`? Seems to be the default Cloudflare response method should be sufficient...or am I missing something? https://developers.cloudflare.com/workers/runtime-apis/response/

Reply:
I could be wrong about `response`, but not 100% sure this can't natively be done from Cloudflare using our existing set of packages